### PR TITLE
updated engines stanza to be >=14

### PR DESCRIPTION
**Pull request by bizob2828**
## Proposed Release Notes
 * Dropped support for Node 12.

## Links
Closes https://issues.newrelic.com/browse/NR-35641

### CODE REVIEW

File: THIRD_PARTY_NOTICES.md [hunk 1]
```diff
@@ -1153,12 +1153,12 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 
 ### tap
 
-This product includes source derived from [tap](https://github.com/tapjs/node-tap) ([v16.0.1](https://github.com/tapjs/node-tap/tree/v16.0.1)), distributed under the [ISC License](https://github.com/tapjs/node-tap/blob/v16.0.1/LICENSE):
+This product includes source derived from [tap](https://github.com/tapjs/node-tap) ([v15.1.6](https://github.com/tapjs/node-tap/tree/v15.1.6)), distributed under the [ISC License](https://github.com/tapjs/node-tap/blob/v15.1.6/LICENSE):
```
Comment (michaelgoin) [inline THIRD_PARTY_NOTICES.md]: Looks like you have an outdated package-lock... your PR is downgrading the version in the license acknowledgements lower than what is actually referenced.

File: package.json [hunk 2]
```diff
@@ -52,7 +52,7 @@
     "tap": "^16.0.1"
   },
   "peerDependencies": {
-    "newrelic": ">=6.11.0"
+    "newrelic": ">=8.15.0"
```
Comment (michaelgoin) [inline package.json]: Whats the driver for bumping this to 8.15?

**Review by michaelgoin — CHANGES_REQUESTED** [review]
Looks like the auto-gen license is off so request changes here. I'm fine if you wanna just rebase after resolving.

File: package.json (hunk 2 above)
Comment (bizob2828) [reply package.json]: meh, saw maurice was doing it. ill just revert and when we ship the agent we can bump to 9.x

Comment (michaelgoin) [reply package.json]: ok. we often don't bump cause it is only important when the API changes. you could argue bumping agent version should require another semver major bump.

Comment (bizob2828) [reply package.json]: oh ok. but don't we want 9.x because we'll drop support for older node versions?

Comment (michaelgoin) [reply package.json]: it won't really matter cause that will be handled by the engines field. you'll notice many of these are on 6.x. but up to ya'll.

Comment (mrickard) [reply package.json]: Bob had suggested on my AWS SDK module PR to bump the peer dependency. If we're not doing this, that's fine. (I'm admittedly not familiar with the automation the team has in place.)

Comment (bizob2828) [reply package.json]: that was different for that PR because you needed a minimum version to remove that code you had done

**Review by michaelgoin — APPROVED** [review]
👍

**Pull request merged**